### ISSUE TITLE
`getValueForSummary` for base options fields should return label instead of value

### ISSUE THREAD

**Issue by gglnx** [verbb/formie#748]
**What are you trying to do?**

Using a summary field in a form with a dropdown field. In the summary instead the label the value is shown. My values are technically and don't match the human-readable labels.

**What's your proposed solution?**

`getValueForSummary` should return the label instead of the value for all base option fields, like dropdowns, radios and checkboxes.

**Comment by engram-design** [comment]
Agreed. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.9](https://github.com/verbb/formie/releases/tag/1.5.9)
